README.md

```diff
@@ -33,22 +32,20 @@ https://cloud.ibm.com/docs/satellite?topic=satellite-getting-started
   </tr>
 </table>
 
-
 ## Compatibility
 
 This module is meant for use with Terraform 0.13 or later.
 
 ## Note
 
-* `location` module creates new location or use existing location ID/name. If user pass the location which is already exist,   satellite-location module will error out and exit the module. In such cases user has to set `is_location_exist` value to true. So that module will use existing location for processing.
-* All optional fields are given value `null` in varaible.tf file. User can configure the same by overwriting with appropriate values.
-* 'satellite-location' module download attach host script in the home directory and appends respective permissions to the script.
-* The modified script must be used in the `user_data` attribute of VSI instance
-* If we want to make use of a particular version of module, then set the argument "version" to respective module version.
+- The `location` submodule creates a location or uses an existing location ID or name. If you pass an existing location, the submodule produces an error and exits the module. To fix the issue, set `is_location_exist` to true so that the module uses the existing location.
```

Comment:
i'm not sure why they originally wrote this from the standpoint of how to make it error. Up to you, but we could just state the two different possibilities: new location or use existing.